ClinGen/clincoded
# Connects to #1372. Connects to #1383. Connects to #1384. Connects to #1392.

**Technical note:**
There is no browser behavior test for #1383 since it is just a misspelling in the internal source code.

**Steps to test #1372:**
1. Create a new GCI Experimental curation.
2. Select the **Expression** option for _Experiment type_.
3. Select either the **A** or **B** options for _Please select which one (A or B) you would like to curate_.
4. In the **Expression** section panel, expect to see that **OLS** is no longer an external link and the **(Ontology Lookup Service)** string is removed in the _Search the Uberon using the OLS (Ontology Lookup Service)._ text.
![image](https://user-images.githubusercontent.com/6956310/28582683-24ba21ee-711b-11e7-8948-3fdb424a0d07.png)
5. Select the **Functional Alteration** option for _Experiment type_.
6. In the **Functional Alteration** section panel, select either the **Patient cells** or **Engineered equivalent** options for _Patient cells with candidate mutation or engineered equivalent?_.
7. Expect to see that **OLS** is no longer an external link and the **(Ontology Lookup Service)** string is removed in the _Search the Cell Ontology (CL) using the OLS (Ontology Lookup Service)._ or the _Search the EFO or Cell Ontology (CL) using the OLS (Ontology Lookup Service)._ text.
![image](https://user-images.githubusercontent.com/6956310/28582917-11948bda-711c-11e7-9dba-9a0f4b91e9b1.png)
8. Repeat steps 6 & 7 for the **Model Systems** and **Rescue** _Experiment type_.
---
**Steps to test #1384:**
1. Go to the `^/create-gene-disease/` page and click on the `Disease+` button.
2. In the **Add Disease** modal, select the _free text_ checkbox at the bottom of the modal.
3. Expect to see **Search HPO-Browser** with an external link beneath the _Phenotype(s) (HPO ID(s))_ text.
![image](https://user-images.githubusercontent.com/6956310/28583413-ec938a3c-711d-11e7-9bee-168ca86b6522.png)
4. Click on that external link and expect to see a new tab/window opened with the **http://compbio.charite.de/hpoweb/showterm?id=HP:0000118** URL.
---
**Steps to test #1392:**
1. Deploy a test instance with the `test_gene_and_disease_data` option.
2. Expect to see the new gene data in the following examples:
`^/genes/EEF1AKMT4/`
`^/genes/TEDC1/`
`^/genes/SOX1-OT/`
